latex.bzl

```diff
@@ -90,20 +90,23 @@ def latex_document(name, main, srcs = [], tags = [], cmd_flags = [], format="pdf
         format = format,
     )
 
-    if "pdf" in format:
-         # Convenience rule for viewing PDFs.
-         native.sh_binary(
-             name = "{}_view_output".format(name),
-             srcs = ["@bazel_latex//:view_pdf.sh"],
-             data = [":{}".format(name)],
-             tags = tags,
-         )
+     # Convenience rule for viewing outputs.
+    native.sh_binary(
+        name = "{}_view_output".format(name),
+        srcs = ["@bazel_latex//:view_output.sh"],
+        data = [":{}.{}".format(name, format)],
+        args = ["$(location :{}.{})".format(name, format)],
```

Comment:
file format shouldn't be needed here right? We Ask only for the single rule output, Bazel should manage to pull that from DefaultInfo

Reply:
Same should apply for the data field

Reply:
It _should_ indeed not be needed, but there's actually a bug with `sh_binary` targets. https://github.com/bazelbuild/bazel/issues/1147 has the details - the `sh_library` workaround works, but we still need to specify the `data` attribute to use the location` helper in `args`, so that's quite a lot more verbose. I don't really have a strong opinion on what the best option is, but I feel like pointing to a specific output file is probably acceptable here.

Reply:
Agrér